frontend/public/components/utils/owner-references.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { K8sResourceKind, OwnerReference, referenceForOwnerRef, } from '../../module/k8s';
+import { ResourceLink } from './index';
+
+export const OwnerReferences: React.FC<OwnerReferencesProps> = ({ resource }) => {
+  const owners = (_.get(resource.metadata, 'ownerReferences') || [])
+    .map((o: OwnerReference) => <ResourceLink key={o.uid} kind={referenceForOwnerRef(o)} name={o.name} namespace={resource.metadata.namespace} />);
+  return owners.length ? <span>{ owners }</span> : <span className="text-muted">No owner</span>;
```

Comment:
I'd think we'd want a `div` here so they're block elements and they stack

```suggestion
  return owners.length ? <div>{ owners }</div> : <span className="text-muted">No owner</span>;
```

Reply:
Actually my suggestion is wrong... Do you know if `ResourceLink` is a block element? Then it should be OK. You could use a fragment here.

```suggestion
  return owners.length ? <>{ owners }</> : <span className="text-muted">No owner</span>;
```

Reply:
Yes, it is. ResouceLink style have a display property flex -block-level" container.